default.nix

```diff
@@ -16,8 +16,6 @@ let
   iohk-ops-extra-runtime-deps = [
     pkgs.git pkgs.nix-prefetch-scripts compiler.yaml
     pkgs.wget pkgs.awscli # for scripts/aws.hs
-    pkgs.nodejs           # for cardano-sl/scripts/js/genesis-hash.js
```

Comment:
What is the new way to perform the manual routine, without `npm` being available?

Reply:
I don't see we need to run this as part of iohk-ops anymore, but we could nixify the upstream